docs/_09_development/_develop_guide/_develop_guide/_develop_new_pkg.rst

```diff
@@ -0,0 +1,320 @@
+.. _development_guide_new_pkg:
+
+------------------------
+Developing a New Package
+------------------------
+
+TDB
+
+
+
+.. _development_guide_new_pkg_architecture:
+
+Architecture of a New Package
+=============================
+
+Every new package should have the following architecture:
+
+.. code-block::
+
+    /package_name
+    /include
+        /package_name
+        header_files(.hpp)
+    /launch
+    /src
+        implementation_files(.cpp)
+        main_files(.cpp)
+    /tests
+        CMakeLists.txt
+        test_files(.cpp)
+    CmakeLists.txt
+    package.xml
+    Readme.md
+    .gitignore
+
+
+
+.. _development_guide_new_pkg_code_style:
+
+Code Style
+==========
+
+Aerostack2 follows the `ROS 2 code style and language versions <https://docs.ros.org/en/humble/The-ROS2-Project/Contributing/Code-Style-Language-Versions.html>`_.
+
+
+
+.. _development_guide_new_pkg_code_style_c++:
+
+C++
+---
+
+Code should follows the `google c++ style guide <https://google.github.io/styleguide/cppguide.html>`_.
```

Comment:
We use ROS 2 code style, but while ROS 2 says it uses Google code style, they use snake_case for functions, while Google use camelCase. We use camelCase

Reply:
Is this the only difference in code style or should I mention that there are a few more?

Reply:
ROS 2 code style says: "Google style guide says CamelCase, but the C++ std library’s style of snake_case is also allowed"
We use Google style (that affects mainly to functions and methods)